aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -8,41 +8,33 @@ By [Peter Kellner](http://peterkellner.net)
 
 The ```AnchorTagHelper``` enhances the html anchor (`<a ... ></a>`) tag. A new set of attributes are defined that work with the anchor tag. The link generated (on the href tag) is based on a combination of these new attributes that work together to form the final redirect URL. That URL can include an optional protocol such as https.
```

Comment:
* Not that it necessarily hurts anything, but three backticks for codeblocks that break across lines and single backticks for in-text code.
* "redirect URL" has special meaning ... I think just "URL" there.
* This passage is a bit confusing. You need prior knowledge or to perhaps read the whole doc to understand that non-standard attributes are converted into standard attributes. I'd refactor this into a complete explanation, and it might even go into the parent document, since it pertains to many of the Tag Helpers.